app/retail/views.py

```diff
@@ -742,84 +744,62 @@ def results(request, keyword=None):
     context['avatar_url'] = static('v2/images/results_preview.gif')
     return TemplateResponse(request, 'results.html', context)
 
-def get_specific_activities(what, trending_only, user, after_pk, request=None, page=None, page_size=10):
+def get_specific_activities(what, trending_only, user, after_pk, request=None, page=1, page_size=10):
```

Comment:
Are we sure setting pagination here wont limit any queries where we're expecting more results to be returned? Just wanting to sanity check it!

Reply:
well without pagination we fetch the entire results which we want to avoid.
I've done a sanity check and it looks like there isn't
I say let's get this in and then wait and see if something breaks  -> we fix to ensure we get paginated requests always